viewers/wicket/model/src/main/java/org/apache/isis/viewer/wicket/model/models/ScalarModel.java

```diff
@@ -349,9 +360,15 @@ public final boolean hasAssociatedActionWithInlineAsIfEdit() {
         if(!spec.isValue()) {
             return Optional.empty();
         }
-        return spec.lookupFacet(ValueFacet.class)
-        .<ObjectAction>flatMap(valueFacet->
-            valueFacet.selectCompositeValueMixinForFeature(getMetaModel()));
+        return getSpecialization().<Optional<ObjectAction>>fold(
+                param->{
+                    return Optional.empty(); //XXX add support later
+                },
+                prop->{
+                    return spec.lookupFacet(ValueFacet.class)
+                            .<ObjectAction>flatMap(valueFacet->
+                                valueFacet.selectCompositeValueMixinForFeature(prop.getManagedProperty()));
```

Comment:
maybe move that logic directly into the ObjectSpecification - or the ManagedProperty